scripts/build.js

```diff
@@ -37,3 +37,11 @@ exec(`./node_modules/.bin/rollup -c scripts/config.js -f es -o dist/${packageNam
 console.log("\nBuilding CommonJS modules...");
 
 exec(`./node_modules/.bin/rollup -c scripts/config.js -f cjs -o dist/${packageName}.js`);
+
+console.log("\nBuilding UMD modules...");
+
+exec(`./node_modules/.bin/rollup -c scripts/config.js -f umd -o dist/${packageName}.umd.js --name ${umdName}`, {EXTERNALS: 'forBrowsers'});
+
+console.log("\nBuilding SystemJS modules...");
+
+exec(`./node_modules/.bin/rollup -c scripts/config.js -f system -o dist/${packageName}.system.js`, {EXTERNALS: 'forBrowsers'});
```

Comment:
I tested the UMD build and saw a "process is not defined" error. Looks like one of the dependencies needs `process.env.NODE_ENV` replaced - I think it's from the `react-is` package.

This change fixes it and also enables minification:

```suggestion
exec(`./node_modules/.bin/rollup -c scripts/config.js -f umd -o dist/${packageName}.umd.js --name ${umdName}`, {EXTERNALS: 'forBrowsers', BUILD_ENV: 'production' });

console.log("\nBuilding SystemJS modules...");

exec(`./node_modules/.bin/rollup -c scripts/config.js -f system -o dist/${packageName}.system.js`, {EXTERNALS: 'forBrowsers', BUILD_ENV: 'production' });
```

@frehner what do you think? Do you want minification on the UMD and SystemJS bundles?

Reply:
great catch. yeah I think it makes sense for the umd+system builds to be minified automatically as well.